gyronimo/metrics/morphism_helena.hh

```diff
@@ -20,27 +20,36 @@
 #ifndef GYRONIMO_MORPHISM_HELENA
 #define GYRONIMO_MORPHISM_HELENA
 
+#include <gsl/gsl_vector.h>
+#include <gsl/gsl_multiroots.h>
+#include <numbers>
+#include <gyronimo/core/error.hh>
+#include <gyronimo/core/multiroot.hh>
 #include <gyronimo/metrics/morphism.hh>
 #include <gyronimo/parsers/parser_helena.hh>
 #include <gyronimo/interpolators/interpolator2d.hh>
 
+
 namespace gyronimo {
 
 //! Morphism from `HELENA` curvilinear coordinates.
 class morphism_helena : public morphism {
- public:
-  morphism_helena(
-      const parser_helena *parser, const interpolator2d_factory *ifactory);
-  virtual ~morphism_helena() override;
-  virtual IR3 operator()(const IR3& q) const override;
-  virtual IR3 inverse(const IR3& x) const override;
-  virtual dIR3 del(const IR3& q) const override;
-  const parser_helena* parser() const {return parser_;};
- private:
-  const parser_helena *parser_;
-  interpolator2d *R_, *z_;
-  static double reduce_2pi(double x);
-  static std::tuple<double, double> reflection_past_axis(double s, double chi);
+public:
```

Comment:
Most of the replacements listed here are just the same lines but with a different number of spaces. So, there are almost no changes, but git lists a lot of them. This practice should be avoided in order to focus on the real changes.